qiskit_metal/tests/test_toolbox_metal.py

```diff
@@ -256,6 +293,11 @@ def test_toolbox_metal_parse_options(self):
                                           expected[x][i],
                                           rel_tol=1e-3)
 
+    def test_toolbox_metal_extract_value_unit(self):
+        """Test functionality of extract_value_unit in toolbox_metal.py."""
+        self.assertEqual(parsing.extract_value_unit("200", "mm"), 200.0)
```

Comment:
Think you would want the testing to be something like

`self.assertEqual(parsing.extract_value_unit("200mm", "mm"), 200.0)`

or

`self.assertEqual(parsing.extract_value_unit("200um", "mm"), 0.2)`